File: Environments/Java11/moderation.java
import java.net.URI;
import java.net.http.HttpClient;
import java.net.http.HttpRequest;
import java.net.http.HttpResponse;
import java.net.http.HttpRequest.BodyPublishers;

public class SkunkOAIClient {
    public static void main(String[] args) throws Exception {
        String url = "https://sp-api.netlify.app/.netlify/functions/moderate-message";
        String json = "{\"prompt_moderation\": \"Your message here\"}";

        HttpClient client = HttpClient.newHttpClient();

        HttpRequest request = HttpRequest.newBuilder()
            .uri(URI.create(url))
            .header("Content-Type", "application/json")
            .POST(BodyPublishers.ofString(json))
            .build();

        HttpResponse<String> response = client.send(request, HttpResponse.BodyHandlers.ofString());

        System.out.println("Response status code: " + response.statusCode());
        System.out.println("Response body: " + response.body());
    }
}

File: Environments/Laravel.PHP/moderation.php
use Illuminate\Support\Facades\Http;

public function moderateMessage(Request $request)
{
    $response = Http::post('https://sp-api.netlify.app/.netlify/functions/moderate-message', [
        'prompt_moderation' => $request->input('prompt_moderation', 'Your message here')
    ]);

    if ($response->successful()) {
        return response()->json([
            'status' => 'ok',
            'data' => $response->json()
        ]);
    } else {
        return response()->json([
            'status' => 'error',
            'message' => $response->body()
        ], $response->status());
    }
}
